aws-test/tests/aws_elastic_beanstalk_application/test-get-query.sql

```diff
@@ -0,0 +1,3 @@
+select arn , name , tags
```

Comment:
```suggestion
select arn, name, tags
```
Remove extra spaces from all the below queries as well.